examples/dataset_generation/create_alpaca_dataset.ipynb

```diff
@@ -8,106 +8,97 @@
     "# Generate dataset for ALPACA\n",
     "\n",
     "## Use token from OpenAI to generate dataset for finetuning\n",
-    "### You can get token from [here](https://beta.openai.com/docs/api-reference/authentication)"
+    "### You can get token from [here](https://beta.openai.com/docs/api-reference/authentication)\n",
```

Comment:
@StochasticRomanAgeev I want to see example in the same notebook of how to use AI 21 and Cohere as well. It is not clear. You can comment out the code if you want and only keep OpenAI by default but the examples need to show that